src/status_im2/contexts/wallet/send/transaction_progress/view.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im2.contexts.wallet.send.transaction-progress.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im2.common.floating-button-page.view :as floating-button-page]
+    [status-im2.common.resources :as resources]
+    [status-im2.contexts.wallet.send.transaction-progress.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn titles
+  [status]
+  (case status
+    :sending   (i18n/label :t/sending...)
+    :confirmed (i18n/label :t/transaction-confirmed)
+    :finalised (i18n/label :t/transacation-finalised)))
```

Comment:
The `case` doesn't have a default value. Otherwise, it will crash if it receives other values.

Reply:
will add 👍